docker-compose.yml

```diff
@@ -108,13 +108,13 @@ services:
       - SETTINGS__BULKRAX__ENABLED=false
       - SETTINGS__FITS_PATH=/opt/fits/fits.sh
       # Comment out these 5 for single tenancy / Uncomment for multi
-      - SETTINGS__MULTITENANCY__ADMIN_HOST=hyku.docker
+      - SETTINGS__MULTITENANCY__ADMIN_HOST=hyku.test
       - SETTINGS__MULTITENANCY__ADMIN_ONLY_TENANT_CREATION=false
-      - SETTINGS__MULTITENANCY__DEFAULT_HOST=%{tenant}.hyku.docker
-      - SETTINGS__MULTITENANCY__ROOT_HOST=hyku.docker
-      - SETTINGS__MULTITENANCY__ENABLED=true
+      - SETTINGS__MULTITENANCY__DEFAULT_HOST=%{tenant}.hyku.test
+      - SETTINGS__MULTITENANCY__ROOT_HOST=hyku.test
+      - SETTINGS__MULTITENANCY__ENABLED=false
```

Comment:
Can you explain the switch to disable multi-tenancy by default?  Does it make sense to remove the next three lines (116-118)?

Reply:
I was testing single tenant mode to make sure the fix i did for the db chicken/egg issue worked. I don't think single should be the default